src/queryLimit.test.js

```diff
@@ -0,0 +1,7 @@
+import queryLimit from './queryLimit';
+
+describe('queryLimit', () => {
```

Comment:
Not for here, but noting that we should ignore these globals such as `describe`

Reply:
@ryandberger , I think #1188 will take care of that.